src/index.html

```diff
@@ -4,11 +4,226 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+<style>
+
+h1 {
+font-size: 80px; 
+text-align: center;
+}
+
+h2 {
+  text-align: left;
+}
+
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+img {
+  opacity: 1.0;
+  filter: alpha(opacity=100); /* For IE8 and earlier */
+}
+
+img:hover {
+  opacity: 0.5;
+  filter: alpha(opacity=60); /* For IE8 and earlier */
+}
+.row .col {
+  float: left;
+  width: 13%;
+  border: 10px solid #fff;
+}
+
+
+table,td,th
+  {
+  border:0px solid black;
+  }
+
+table
+  {
+  width:100%;
+  }
+
+th
+  {
+  height:50px;
+  }
+td
+{
+text-align:left;
+}
+label{float: left;width: 80px;line-height:25px;} 
+form{margin:0px} 
+input{width: 180px;border:1px solid #808080;} 
+textarea{width: 250px;height: 150px;} 
+.main{
+ text-align: center;
+border-radius: 20px;
+width: 1500px;
+height: 350px;
+margin: auto;
+position: absolute;
+  top: 0;
+left: 0;
+ right: 0;
+ bottom: 0;
+}
+</style>
   </head>
   <body>
-    <div id="main"></div>
+    <div class="main">
+    
     <script src="./js/index.js"></script>
+    <h1>John Roberts</h1>
+    <h2>Home</h2>
+    <button style="margin-left: 1050px">DOWNLOAD MY FULL RESUME</button>
+    <img SRC="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_248,h_244,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.webp"  class="img-circle" alt="Jhon">
+    <p style="font-size:20pt;color:DodgerBlue;text-align:center">BUSINESS EXPERT<p>
```

Comment:
Might consider to move the styles at least to the header styles section. 
Inline style is not recommended due to hard to override later on.

Reply:
Move to the index.scss, but for other problem is if I used <p></p> other text, they will all follow this style. How to write only for specifying text？ Thank you.

Reply:
Styling is not within the scope of this homework.

Nick will be talking about CSS selectors in the next lessons. So don't worry too much about the CSS part of things for now.

Reply:
OK， thank you.